### PR TITLE
[1LP][RFR] Fix test_cloud_events for changes in 5.10 vs 5.9

### PR DESCRIPTION
**Pull request by john-dupuy**
Purpose or Intent
=================

__Fixing__ both tests in `test_cloud_events` for CFME 5.10. An error was occuring in the `cmp_func` for these events. The data returned for the event does not have `status` or `subStatus` in CFME 5.10. 

{{ pytest: --long-running --use-provider azure cfme/tests/cloud/test_cloud_events.py }}

Exception that was seen: 
```
2019-01-23 14:21:05,143 [I] [cfme] (events) [(BaseEvent(source=AZURE, full_data=will be ignored, event_type=networkSecurityGroups_write_EndRequest), 0), (BaseEvent(source=AZURE, full_data=will be ignored, event_type=networkSecurityGroups_delete_EndRequest), 0)] (cfme/utils/events.py:270)
2019-01-23 14:21:05,406 [I] [cfme] (rest-api) [RESTAPI] GET https://10.16.5.92/api/event_streams/919 {} (/home/jdupuy/cfme_tests/cfme_venv/lib/python2.7/site-packages/manageiq_client/api.py:109)
2019-01-23 14:21:05,714 [E] [cfme] (events) An exception during matching events occurred. (cfme/utils/events.py:240)
Traceback (most recent call last):
  File "/home/jdupuy/cfme_tests/integration_tests/cfme/utils/events.py", line 234, in process_events
    if exp_event['event'].matches(got_event):
  File "/home/jdupuy/cfme_tests/integration_tests/cfme/utils/events.py", line 103, in matches
    if not self.event_attrs[attr].match(evt.event_attrs[attr]):
  File "/home/jdupuy/cfme_tests/integration_tests/cfme/utils/events.py", line 37, in match
    return self.cmp_func(self.value, attr.value)
  File "/home/jdupuy/cfme_tests/integration_tests/cfme/tests/cloud/test_cloud_events.py", line 57, in rm_cmp
    return data['resourceId'].endswith(nsg_name) and data['status']['value'] == 'Succeeded'
KeyError: 'status'```

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 0e4bd3d54444ea3eeae59a8a506aea9655675775:

`cfme/tests/cloud/test_cloud_events.py`:
- :bangbang: Line 46:21: **E126** *continuation line over-indented for hanging indent*
- :bangbang: Line 118:9: **E303** *too many blank lines (2)*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by izapolsk** [comment]
:+1:, LGTM

**Comment by dajoRH** [comment]
Lint report for commit fcebdf8ca799e2bab2511411700533b893b51781:

`cfme/tests/cloud/test_cloud_events.py`:
- :bangbang: Line 46:22: **E126** *continuation line over-indented for hanging indent*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/tests/cloud/test_cloud_events.py [hunk 1]
```diff
@@ -54,7 +64,13 @@ def rm_cmp(_, y):
         # In 5.9 version `y` is a dict, not a yaml stream.
         data = yaml.safe_load(y) if appliance.version < '5.9' else y
```
Comment (ganeshhubale) [inline cfme/tests/cloud/test_cloud_events.py]: Please remove check: ```if appliance.version < 5.9```

File: cfme/tests/cloud/test_cloud_events.py [hunk 2]
```diff
@@ -96,7 +112,13 @@ def cmp_function(_, y):
...
```
Comment (ganeshhubale) [inline cfme/tests/cloud/test_cloud_events.py]: Same here for ```< 5.9``` check.

File: cfme/tests/cloud/test_cloud_events.py [hunk 3]
```diff
@@ -38,10 +38,20 @@ def add_cmp(_, y):
...
```
Comment (ganeshhubale) [inline cfme/tests/cloud/test_cloud_events.py]: Please remove  ```< 5.9``` check.

File: cfme/tests/cloud/test_cloud_events.py (hunk 1 above)
Comment (john-dupuy) [reply cfme/tests/cloud/test_cloud_events.py]: @ganeshhubale, will do!

**Comment by dajoRH** [comment]
Lint report for commit 76a7229e912824fb6310e9bd59b6b979fec458be:

`cfme/tests/cloud/test_cloud_events.py`:
- :red_circle: Line 4:1: **F401** *'yaml' imported but unused*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by jawatts** [comment]
PR failing isort :crying_cat_face:

**Comment by jawatts** [comment]
Restarted travis due to failure on isort. Waiting on it to finish before moving to 1LP

**Comment by john-dupuy** [comment]
Moving to WIPTEST as this is impacted by yaml MR 691

**Comment by izapolsk** [comment]
I think PR errors should disappear after https://github.com/ManageIQ/wrapanapi/pull/359

**Comment by izapolsk** [comment]
I added my wrapanapi patch to run tests against it.

**Pull request merged**